tests/test_spyre_online.py

```diff
@@ -0,0 +1,173 @@
+import os
+import subprocess
+import sys
+import time
+from typing import Optional
+
+import openai
+import pytest
+import requests
+from vllm.entrypoints.openai.cli_args import make_arg_parser
+from vllm.utils import FlexibleArgumentParser, get_open_port
+
+from tests.spyre_util import get_spyre_backend_list, get_spyre_model_list
+
+
+class RemoteOpenAIServer:
```

Comment:
I think it is fine for now, but I'd like there to be a better way to use some of vLLM's test utilities instead of copying. Maybe the vllm test framework/utilities could become its own package for use in plugins.

Reply:
👍  yeah that would be super cool

Reply:
@joerunde In my PR #59, I moved this class to spyre_util.py, I wondered if it could be reused until we can make better use of vLLM's utils like Travis suggested.

Reply:
yup, just rebased this to make use of that. Thanks!